packages/ui-library/src/components/radio/index.ts

```diff
@@ -96,14 +122,15 @@ export class BlrRadio extends LitElementCustom {
             `
           : nothing}
       `;
-
+      const id = calculateOptionId(this.label);
       return html`
         <div class="blr-radio ${classes}">
           <input
-            id=${this.optionId || nothing}
+            id=${id || nothing}
             class="${classes} input-control"
             type="radio"
             name=${this.name}
+            .value=${this.value}
```

Comment:
in addition to the value we should probably also set `checked` property of the native input element.
my guess is that this is what prevented correctly updating the element state when the radio's custom `checked` property changes.

`.checked=${this.checked}`

line 137 only sets the html-attribute which is only used during element initialisation and will not update the input state if changed.

Reply:
Same reason as above, implementing with .checked=${this.checked} still does not trigger any method when checked is changed from developer console.